go.mod

```diff
@@ -1,15 +1,33 @@
 module github.com/aoscloud/aos-core-rcar-gen4
 
-go 1.14
+go 1.18
```

Comment:
Let's stick with one format of config message: either `tag: message...` or `[tag] Message`.